iscsi/get.go

```diff
@@ -183,12 +181,14 @@ func (iblock IBLOCK) GetIblockUdev(targetCorePath string, iblockNumber string,
 
 // GetRBDMatch is getting the actual info to build up
 // the RBD data and match with the enable target
-func (rbd RBD) GetRBDMatch(sysDevicePath string, rbdNumber string, poolImage string) (r *RBD, err error) {
-
-	rbd.Name = "rbd_" + rbdNumber
-	rbd.Rnumber = rbdNumber
-
-	systemRbds, err := filepath.Glob(filepath.Join(sysDevicePath, "/devices/rbd/[0-9]*"))
+func (fs FS) GetRBDMatch(rbdNumber string, poolImage string) (*RBD, error) {
+	rbd := RBD{
+		Name:    "rbd_" + rbdNumber,
+		Rnumber: rbdNumber,
+		Pool:    "",
+		Image:   "",
```

Comment:
Since `Pool` and `Image` are empty strings, you can leave them out of the struct initialization.

Reply:
Sure, I will do that and squashing it down for the PR. Thanks